src/vm/dllimportcallback.cpp

```diff
@@ -1126,11 +1121,10 @@ void UMEntryThunk::Terminate()
 {
     WRAPPER_NO_CONTRACT;
 
-#ifdef FEATURE_WINDOWSPHONE
     SystemDomain::GetGlobalLoaderAllocator()->GetExecutableHeap()->BackoutMem(this, sizeof(UMEntryThunk));
-#else
-    DeleteExecutable(this);
-#endif
+
+    _ASSERTE(SystemDomain::GetGlobalLoaderAllocator()->GetExecutableHeap()->IsRelaxed());
+    FillMemory(&m_code, sizeof(m_code), 0xcc);
```

Comment:
0xCC works well for x86 because of it is code for the breakpoint instruction. What does it do on arm?